Quick-Start-Guide.md

```diff
@@ -16,8 +16,9 @@
     * **Set a root password**.
     * **Use UTF8 as character set**.
     * **IT IS INCREDIBLY IMPORTANT** that you check the `Use UTF8 as server's character set` checkbox on the `Default instance properties` page during installation. If you don't do this you may face very hard to diagnose crashes.
-* Install [Python 3](https://www.python.org/downloads/).
-  * The latest version is fine, during installation check the `add python.exe to PATH` checkbox.
+* Install [Python 3.12](https://www.python.org/downloads/).
+  * We're a bit behind for how we use Python, so [version 3.12](https://www.python.org/downloads/release/python-3127/) is required, other versions break with our usage!
```

Comment:
Is the issue with Python 3.13 on our side, or is it an issue with the mariadb package? I thought it was the latter.

Also it's not necessary to add Python to PATH since our instructions use the py launcher, which is added to the Windows folder by default and should always be accessible (not that it's a bad idea to add it to the PATH, just wanted to note.)